package.json

```diff
@@ -36,8 +36,8 @@
     "karma-mocha": "^0.1.9",
     "karma-phantomjs-launcher": "^0.1.4",
     "mocha": "^1.21.4",
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0",
+    "react": "^15.5.4",
```

Comment:
I made that change because I don't think it's really appropriate to `||` multiple versions together in `devDependencies`.  I only do that in `peerDependencies`.  But it's true, we could leave this change out of this PR.